File: backend/supabase_client.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv()

url: str = os.getenv("SUPABASE_URL", "https://kkgdaxuxdwqzktzanflq.supabase.co")
key: str = os.getenv("SUPABASE_SERVICE_ROLE_KEY", "")

if url and key:
    supabase: Client = create_client(url, key)
else:
    supabase = None

File: src/services/supabaseClient.js
import { createClient } from '@supabase/supabase-js';

const supabaseUrl = import.meta.env.VITE_SUPABASE_URL || 'https://kkgdaxuxdwqzktzanflq.supabase.co';
const supabaseAnonKey = import.meta.env.VITE_SUPABASE_ANON_KEY || 'sb_publishable_8iHhorjXkY4UUUyB1kJe5w_S62_540_';

export const supabase = createClient(supabaseUrl, supabaseAnonKey, {
  auth: {
    persistSession: true,
    autoRefreshToken: true,
    detectSessionInUrl: true,
  },
});
